plumgrid_playbooks/enable_pg_extensions.yml

```diff
@@ -19,16 +19,9 @@
   tasks:
     - name: Remove previous PLUMgrid sources.list
       file:
-        path: /etc/apt/sources.list.d/plumgrid.list
+        path: "/etc/apt/sources.list.d/plumgrid.list"
         state: absent
 
-    - name: Add PLUMgrid openstack to repo
-      lineinfile:
-        dest: /etc/apt/sources.list.d/plumgrid.list
-        line: "deb {{ plumgrid_repo }}/openstack/deb/{{ openstack_version }} openstack-{{ openstack_version }} {{ plumgrid_component }}"
-        state: present
-        create: yes
-
     - name: Copy PLUMgrid GPG-key file
```

Comment:
@syedalijabir is there any reason that you are using different `GPG` for each release?